views/components/Graphs/BarGraph.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { 
+  VictoryBar, 
+  VictoryChart, 
+  VictoryAxis, 
+  VictoryTheme, 
+  VictoryStack } from 'victory';
+
+
+const BarGraph = ({ graph }) => {
+  const tickValues = graph.data.map((_, i) => (i))
+  const tickFormat = graph.data[0].map(({siteName}) => siteName)
```

Comment:
is the site name different for each index in `graph.data`? Maybe I'm confused about the shape of `data`, I was thinking that `data` is an array where each entry is an array subject data for a site. Is that incorrect?

Reply:
All the sites are included in an array based on their variable value. The snippet is what victory wants the shape of the data to be, and the snippet below is the for the sites.
`javascript
const data2012 = [
  {quarter: 1, earnings: 13000},
  {quarter: 2, earnings: 16500},
  {quarter: 3, earnings: 14250},
  {quarter: 4, earnings: 19000}
];

const data2013 = [
  {quarter: 1, earnings: 15000},
  {quarter: 2, earnings: 12500},
  {quarter: 3, earnings: 19500},
  {quarter: 4, earnings: 13000}
];
`
`
[
  [
    { siteName: 'CA', count: 7, fieldLabel: 'Filled Out' },
    { siteName: 'ProNET', count: 2, fieldLabel: 'Filled Out' }
  ],
  [
    { siteName: 'CA', count: 3, fieldLabel: 'Did not Fill out' },
    { siteName: 'ProNET', count: 4, fieldLabel: 'Did not Fill out' }
  ]
]
`
This is the documentation link
https://formidable.com/open-source/victory/docs/#8-stack-multiple-bar-charts